### DISCUSSION TITLE
Bot verify info not registered by any registrars

### DISCUSSION THREAD

**Discussion by HAILANGER** [simple-robot/Discussions#111] [打扰了]
### 涉及的编程语言

Java

### 疑问描述

Bot verify info [love.forte.simbot.bot.PathBotVerifyInfo@53314f76] not registered by any registrars, skip. The botRegistrar: love.forte.simboot.spring.autoconfigure.SimbotSpringBootBotAutoRegisterBuildConfigure$SimbotSpringBootBotAutoRegisterPostProcessor@7d30007d

**Comment by ForteScarlet** [comment]
任何与预期不符、疑似bug或有异常的错误请反馈到 [issues](https://github.com/simple-robot/simpler-robot/issues/new?assignees=ForteScarlet&labels=bug%3F&template=bug-report.yml)

**Reply by b3601993** [reply]
我也遇到这个问题，你解决了吗？
